metrix-integration/src/main/java/com/powsybl/metrix/integration/MetrixComputationType.java

```diff
@@ -14,7 +14,8 @@ public enum MetrixComputationType {
     OPF(0),
     LF(1),
     OPF_WITHOUT_REDISPATCHING(2),
-    UNKNOWN(3);
+    OPF_WITH_OVERLOAD(3),
+    UNKNOWN(4);
```

Comment:
You changed the value linked to "UNKNOWN" (from 3 to 4) but it doesn't seem to have any impact in the tests? Where is it used?